src/C4PlayerInfoListBox.cpp

```diff
@@ -921,21 +921,25 @@ C4GUI::ContextMenu *C4PlayerInfoListBox::ClientListItem::OnContext(C4GUI::Elemen
 	C4Client *pClient = GetClient();
 	// create context menu
 	C4GUI::ContextMenu *pMenu = new C4GUI::ContextMenu();
+	// helper function
+	auto AddMenuItem = [&](const char *text, const char *description, auto callbackFn)
```

Comment:
I am not sure about the (still unpublished) style guidelines, but I prefer explicitly specifying captured variables.

Reply:
Agreed.